File: 2310817120009_AlysaArmelia_PemrogramanI_Modul4/Python/PRAK403-2310817120009-AlysaArmelia.py
bilangan, angka = map(int, input("").split())

p = bilangan
q = angka

while (p < angka and q > bilangan):
    print(f"{p} {q} - ", end="")
    
    p += 1
    q -= 1

    if (p == angka and q == bilangan):
        print(f"{p} {q}")

while (p > angka and q < bilangan):
    print(f"{p} {q} - ", end="")
    
    p -= 1
    q += 1

    if (p == angka and q == bilangan):
        print(f"{p} {q}")

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul4/Bahasa C/PRAK403-2310817120009-AlysaArmelia.c
#include <stdio.h>

int main(){
    int bilangan; 
    int angka;

    scanf("%d %d", &bilangan, &angka);

    int p = bilangan;
    int q = angka;
    
    while(p < angka && q > bilangan)
    {
        printf("%d %d - ", p, q);
        
        p++;
        q--;

        if(p == angka && q == bilangan)
        {
            printf("%d %d ", p, q);
        }
    }
    while(p > angka && q < bilangan)
    {
        printf("%d %d - ", p, q);
        
        p--;
        q++;

        if(p == angka && q == bilangan)
        {
            printf("%d %d ", p, q);
        }
    }
return 0;
}
